content/millennium/r4/individuals/related-person.md

```diff
@@ -111,8 +111,124 @@ List an individual RelatedPerson by its id:
 
 ### Errors
 
+## Create
+
+Create a new Encounter Level RelatedPerson.
+
+    POST /RelatedPerson
+
+_Implementation Notes_
+
+* Only the body fields mentioned below are supported. Unsupported fields will be ignored.
+
+### Authorization Types
+
+<%= authorization_types(provider: true, patient: true, system: true) %>
+
+### Headers
+
+<%= headers head: {Authorization: '&lt;OAuth2 Bearer Token>', 'Content-Type': 'application/fhir+json'} %>
+
+### Body Field
+
+<%= definition_table(:related_person, :create, :r4) %>
+
+### Example
+
+#### Request
+
+    POST https://fhir-ehr-code.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/RelatedPerson
+
+#### Body
+
+<%= json(:r4_relatedperson_create) %>
+
+#### Response
+
+<%= headers status: 201 %>
+<pre class="terminal">
+Cache-Control: no-cache
+Content-Length: 0
+Content-Type: text/html
+Date: Wed, 27 Mar 2019 17:23:14 GMT
+Etag: W/"0"
+Location: https://fhir-ehr.sandboxcerner.com/r4/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/Patient/5786010
+Last-Modified: Wed, 27 Mar 2019 17:23:14 GMT
+Server-Response-Time: 296.405243
+Status: 201 Created
+Vary: Origin
+X-Request-Id: 11111111111111111111111111111111
+X-Runtime: 2.011826
+</pre>
+
+The `ETag` response header indicates the current `If-Match` version to use on subsequent updates.
+
+### Error
+
+The common [errors] and [OperationOutcomes] may be returned.
+
+## Patch
+
+Patch an existing Encounter-level Coverage.
+
+    PATCH /RelatedPerson/:id
+
+_Implementation Notes_
+
+* This implementation follows the [JSON Patch](https://tools.ietf.org/html/rfc6902) spec.
+* Only operations on the paths listed below are supported.
+* Only Encounter-level Coverages may be patched. Patches are not currently supported for Patient-level Coverages.
```

Comment:
'Coverage' -> 'RelatedPerson'

Reply:
updated here: https://github.com/cerner/fhir.cerner.com/pull/527/commits/c5a51de325f428be24ed8e8c510ddcd737ea96c3#diff-e6b1787a35b755b7855af66f7e6243aa4d41f2e1b9c63544e414590074cf99abR172

Reply:
missed a Coverage at the end

Reply:
updated here: https://github.com/cerner/fhir.cerner.com/pull/527/commits/5803ca15b4740206577992e4a5177fece84ff0fb#diff-e6b1787a35b755b7855af66f7e6243aa4d41f2e1b9c63544e414590074cf99abR180